cni/network/network.go

```diff
@@ -1258,18 +1259,25 @@ func convertNnsToCniResult(
 
 			intIndex := i
 			for _, ip := range ni.Ipaddresses {
+				ipAddr := net.ParseIP(ip.Ip)
 
-				ipWithPrefix := fmt.Sprintf("%s/%s", ip.Ip, ip.PrefixLength)
-				_, ipNet, err := net.ParseCIDR(ipWithPrefix)
+				var address net.IPNet
```

Comment:
can we move this above line 1272 (before To4() check)

Reply:
Sure, will move it above 1272